db/schema.rb

```diff
@@ -1160,17 +1160,6 @@
     t.index ["status"], name: "index_vba_documents_upload_submissions_on_status"
   end
 
-  create_table "versions", force: :cascade do |t|
-    t.string "item_type", null: false
-    t.bigint "item_id", null: false
-    t.string "event", null: false
-    t.string "whodunnit"
-    t.text "object"
-    t.datetime "created_at"
-    t.text "object_changes"
-    t.index ["item_type", "item_id"], name: "index_versions_on_item_type_and_item_id"
-  end
-
```

Comment:
The first line of `schema.rb` (`ActiveRecord::Schema[7.1].define(version: 2024_03_18_190237) do`) wasn't updated because there was a migration added after this one was added.